### PR TITLE
Add oragono.io/maxline-2

### PR DESCRIPTION
**Pull request by DanielOaks**
My draft for the new `oragono.io/maxline-2` capability, including some rewritten content and a focus on the 'rest' portion of the IRC message. @slingamn thoughts on this as current written?

[Rendered Document](https://github.com/oragono/oragono-site/blob/maxline-2/maxline-2.md)

### CODE REVIEW

**Comment by slingamn** [comment]
How does this interact with msgid? Our current behavior is that when a message is split, every piece is sent with the same msgid. This is probably wrong.

**Comment by jwheare** [comment]
Yup they’d need their own ids.

File: maxline-2.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+### Interaction with PRIVMSG and NOTICE
+If a client has negotiated the `oragono.io/maxline-2` capability and sends a `PRIVMSG` or a `NOTICE` message that is longer than 512 bytes, the receiving server MUST split this into multiple regular length messages (maximum 512 bytes) when sending it to clients that have not negotiated this capability.
+
+Servers SHOULD split on whitespace or other word boundaries, and may use whatever method is easiest for them to implement. Lines MUST NOT be split in the middle of a UTF-8 character.
+
+Servers MAY split other commands/numerics into multiple lines in whatever way ensures the best compatibility and functionality.
+
+### The `oragono.io/truncated` Tag
+The `oragono.io/truncated` tag, when present, indicates that a message has been truncated due to the client's line length. It may be sent to any client which has negotiated the [`message-tags` capability](https://ircv3.net/specs/extensions/message-tags.html#message-tags-capability).
```
Comment (slingamn) [inline maxline-2.md:43]: Should this "may" be a "MAY" (or a "SHOULD")?

**Comment by DanielOaks** [comment]
Hmm, given the sort of behavior this refers to, it makes sense to leave it
as is I think.

File: maxline-2.md (hunk 1 above)
Comment (DanielOaks) [reply maxline-2.md:43]: Hmm, given the sort of behavior this refers to, it makes sense to leave it as is I think.

**Comment by slingamn** [comment]
This looks good to me as long as we add a note about msgids. Something like: if message IDs are enabled, then the split messages MUST NOT share msgids with each other or with the unsplit message.

**Pull request merged**